test/support/stripe_mock.ex

```diff
@@ -53,14 +53,23 @@ defmodule Stripe.StripeMock do
 
   @impl true
   def handle_info(
-        %{os_pid: os_pid, opts: opts, restarting: {true, from}} = state,
-        {:DOWN, os_pid, :process, _ex_pid, _reason}
+        {:DOWN, os_pid, :process, _ex_pid, _reason},
+        %{os_pid: os_pid, opts: opts, restarting: {true, from}} = state
       ) do
     {:ok, manager_pid, os_pid} = start_stripe_mock(opts)
     GenServer.reply(from, :ok)
     {:noreply, %{state | manager_pid: manager_pid, os_pid: os_pid, restarting: false}}
   end
 
+  @impl true
+  def handle_info(
+        {:DOWN, os_pid, :process, _ex_pid, _reason},
+        %{os_pid: os_pid, opts: opts, restarting: false} = state
+      ) do
+    {:ok, manager_pid, os_pid} = start_stripe_mock(opts)
+    {:noreply, %{state | manager_pid: manager_pid, os_pid: os_pid, restarting: false}}
+  end
+
```

Comment:
Why are these changes needed?

Reply:
They are needed to run the tests, but I guess they could be a separate PR to clean up the repo a bit

Reply:
Yes, please 🙏🏻